internal/controller/server/client_services.go

```diff
@@ -84,9 +84,9 @@ func (s *clientCPServicesServer) GetStreamLogs(request *clientCPproto.RequestFor
 	return nil
 }
 
-//ExecuteJob will call Executejob function of execution and get jobId
-func (s *clientCPServicesServer) ExecuteJob(ctx context.Context, execute *clientCPproto.RequestForExecute) (*clientCPproto.Response, error) {
-	jobId, err := s.procExec.ExecuteJob(ctx, execute.JobName, execute.JobData)
+// ExecuteJob will call ExecuteJob function of execution and get jobId
+func (s *clientCPServicesServer) ExecuteJob(ctx context.Context, executionContext *clientCPproto.RequestForExecute) (*clientCPproto.Response, error) {
```

Comment:
Dont use the name *Context with requestForExecute as context has a very different meaning in golang. I would suggest using a name like request, executionVars, etc

Reply:
It contains all execution-related things like job name, arguments, and some other info, so we gave this structure name as executionContext in all files.

Reply:
I guess its not good to use this name. Please use a different name for this variable

Reply:
we have changed it to executionData